tensorflow/lite/experimental/micro/kernels/BUILD

```diff
@@ -1,518 +1,722 @@
-load("//tensorflow/lite:build_def.bzl", "tflite_copts")
```

Comment:
It looks like the formatting got messed up on this file?

Reply:
You are right. Many thanks for checking. I have just fixed it